websites/W/Wolfram Alpha/presence.ts

```diff
@@ -0,0 +1,211 @@
+const presence = new Presence ({
+    clientId: "826428815365373994"
+}),
+    browsingStamp = Math.floor(Date.now() / 1000),
+    path = document.location.pathname;
+
+presence.on("UpdateData", async () => {
+    const presenceData: PresenceData = {
+        largeImageKey: "logo",
+        startTimestamp: browsingStamp
+    };
+
+    if (path.includes("/input")) {
+        presenceData.details = "Searching for:";
+        presenceData.state = document.querySelector("._9CcbX").textContent;
+        presenceData.smallImageKey = "search";
+    }
+    else if (path.includes("/examples")) {
+        const title = document.getElementById("page-title").textContent;
+
+        presenceData.details = "Viewing Page:";
+        presenceData.state = "Examples page";
+        
+        switch (path.includes("/mathematics")) {
+            case (path.includes("/elementary-math")):
+            case (path.includes("/algebra")):
+            case (path.includes("/calculus-and-analysis")):
+            case (path.includes("/geometry")):
+            case (path.includes("/plotting-and-graphics")):
+            case (path.includes("/differential-equations")):
+            case (path.includes("/numbers")):
+            case (path.includes("/trigonometry")):
+            case (path.includes("/linear-algebra")):
+            case (path.includes("/discrete-mathematics")):
+            case (path.includes("/number-theory")):
+            case (path.includes("/complex analysis")):
+            case (path.includes("/mathematical-function")):
+            case (path.includes("/logic-and-set-theory")):
+            case (path.includes("/mathematical-definitions")):
+            case (path.includes("/continued-fractions")):
+            case (path.includes("/famous-maths-problems")):
+            case (path.includes("/statistics")):
+            case (path.includes("/common-core-maths")):
+            case (path.includes("/probability")):
+                presenceData.details = "Reading for:";
```

Comment:
```suggestion
                presenceData.details = "Reading article:";
```

Reply:
```suggestion
                presenceData.details = "Reading about:";
```

Reply:
Idk, for is not the correct thing at least xD

Reply:
I might change it to "about", it looks good :)